src/Wheel.js

```diff
@@ -1,8 +1,19 @@
 class Wheel {
   constructor(dataFilePath) {
     this.dataFilePath = dataFilePath;
```

Comment:
Is the parameter of `dataFilePath` just the data for the wheel elements, and nothing else? From the naming, it makes me think that you are pulling in _all_ the WOF data. Be sure to not do this... and if you aren't, think about renaming this parameter to avoid confusion.